diff --git a/node_modules/gatsby-transformer-documentationjs/gatsby-node.js b/node_modules/gatsby-transformer-documentationjs/gatsby-node.js
index d9be778..fa2c032 100644
--- a/node_modules/gatsby-transformer-documentationjs/gatsby-node.js
+++ b/node_modules/gatsby-transformer-documentationjs/gatsby-node.js
@@ -1,8 +1,8 @@
-"use strict";
+'use strict';
 
-var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");
+var _interopRequireDefault = require('@babel/runtime/helpers/interopRequireDefault');
 
-var _objectWithoutPropertiesLoose2 = _interopRequireDefault(require("@babel/runtime/helpers/objectWithoutPropertiesLoose"));
+var _objectWithoutPropertiesLoose2 = _interopRequireDefault(require('@babel/runtime/helpers/objectWithoutPropertiesLoose'));
 
 const documentation = require(`documentation`);
 
@@ -202,7 +202,7 @@ exports.onCreateNode = async (_ref) => {
     node,
     actions
   } = _ref,
-      helpers = (0, _objectWithoutPropertiesLoose2.default)(_ref, ["node", "actions"]);
+      helpers = (0, _objectWithoutPropertiesLoose2.default)(_ref, ['node', 'actions']);
   const {
     createNodeId,
     createContentDigest
@@ -290,7 +290,7 @@ exports.onCreateNode = async (_ref) => {
       const docSkeletonNode = {
         commentNumber,
         level,
-        id: createNodeId(docId(node.id, docsJson)),
+        id: createNodeId(docId(parent, docsJson)),
         parent,
         children: [],
         internal: {
